ui/src/me/components/GettingStarted.tsx

```diff
@@ -1,53 +1,120 @@
 // Libraries
-import React, {PureComponent} from 'react'
-import {withRouter, Link, WithRouterProps} from 'react-router'
+import React, {FunctionComponent, useState} from 'react'
+import {withRouter, WithRouterProps} from 'react-router'
 
 // Components
-import GradientBorder from 'src/shared/components/cells/GradientBorder'
+import {
+  Panel,
+  Button,
+  Gradients,
+  InfluxColors,
+  ComponentColor,
+  ComponentSize,
+} from '@influxdata/clockface'
+import CollectorGraphic from 'src/me/graphics/CollectorGraphic'
 import DashboardingGraphic from 'src/me/graphics/DashboardingGraphic'
 import ExploreGraphic from 'src/me/graphics/ExploreGraphic'
-import CollectorGraphic from 'src/me/graphics/CollectorGraphic'
 
-class GettingStarted extends PureComponent<WithRouterProps> {
-  public render() {
-    const {
-      params: {orgID},
-    } = this.props
-    return (
-      <div className="getting-started">
-        <div className="getting-started--container">
-          <Link
-            to={`/orgs/${orgID}/load-data/telegrafs`}
-            className="getting-started--card"
-          >
-            <GradientBorder />
-            <CollectorGraphic />
-            <h3 className="getting-started--title">Load your data</h3>
-          </Link>
-        </div>
-        <div className="getting-started--container">
-          <Link
-            to={`/orgs/${orgID}/dashboards`}
-            className="getting-started--card"
-          >
-            <GradientBorder />
-            <DashboardingGraphic />
-            <h3 className="getting-started--title">Build a dashboard</h3>
-          </Link>
-        </div>
-        <div className="getting-started--container">
-          <Link
-            to={`/orgs/${orgID}/alerting`}
-            className="getting-started--card"
-          >
-            <GradientBorder />
-            <ExploreGraphic />
-            <h3 className="getting-started--title">Set up alerting</h3>
-          </Link>
-        </div>
-      </div>
-    )
+const GettingStarted: FunctionComponent<WithRouterProps> = ({
+  params: {orgID},
```

Comment:
it would be super cool if this came from the org store and not the router. feels weird, because we are including the router anyways, but by having redux be the single source of truth and only using the router as an event marker / generator, it makes the code easier to manage overall

Reply:
That's a good call, I'll update this guy. We gon' catch all them loose threads